proj/hps_accel/Makefile

```diff
@@ -52,6 +52,9 @@ DEFINES += INCLUDE_MODEL_HPS
 RUN_MENU_ITEMS=1 1 c
 TEST_MENU_ITEMS=3 a
 
+# Ensure perf_csrs logic is present
+DEFINES += CONFIG_CPU_PERF_CSRS
```

Comment:
Hmm, I thought Litex was supposed to define this if the perf CSRs are implemented in our design? If that's not happening, we should fix Litex -- or possibly we don't actually have the perf CSRs :-)

Reply:
The setting of CONFIG_CPU_PERF_CSRS is in a WIP PR I have.   It will be added as a monkey patch applied to LiteX.   So yes, this needs to be done for now, and can / should be removed once that PR lands (#253).